src/models/ComponentAttribute.js

```diff
@@ -91,17 +91,31 @@ class ComponentAttribute {
    */
   __typeOfValueValidation() {
     const type = this.type.toLowerCase();
-
-    if (this.definition.type === 'Link') {
-      return !Array.isArray(this.value) || this.value.some((link) => typeof link !== 'string');
-    }
+    let typeofvalue = typeof this.value;
```

Comment:
why don't you need esilnt disable here ?

Reply:
There was no eslint error so I didnt use it.